### ISSUE TITLE
OAuth 2.0 support

### ISSUE THREAD

**Issue by wyattbenno777** [pipedrive/client-nodejs#78]
Hello.

Does this support oauth tokens instead of api token?
Pipedrive only allows marketing support for projects that use the oauth token.

If this exists can it be added into the docs??

**Comment by kmuenze** [comment]
yes - this would be very helpful.

**Comment by kmuenze** [comment]
Found the pull request: https://github.com/pipedrive/client-nodejs/pull/71

Looks like it is close!

**Comment by kmuenze** [comment]
Is this going to get released?

**Comment by tot-ra** [comment]
@kmuenze pending on @ziimk

**Comment by kmuenze** [comment]
@tot-ra That is unfortunate. Maybe you could fork the code and then people could use that instead. It looks like dev on the official library has basically stopped.

**Comment by eerootsus** [comment]
I have tweaked https://github.com/pipedrive/client-nodejs/pull/71 code a bit and once the author approves and merges, I believe we can merge into here as well.
